_projects/2018/astropy/mcmc.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Add seamlessly compatibility between astropy.modeling.models and emcee
```

Comment:
The name should be shorter and more catchy as this will be the title in the ideas page